starter-code/main.js

```diff
@@ -53,7 +53,187 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "7":
+            //List the 10 companies with more employees sorted ascending (show name and employees).
+            db.collection('companies').find().sort({number_of_employees : -1}).limit(10).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "8":
+            //Find the company with the name "Facebook"
+            db.collection('companies').find({name : "Facebook"}, {name: 1, _id: 0}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "9":
+            //How many employees has Facebook?
+            db.collection('companies').find({name : "Facebook"}, {number_of_employees: 1, name: 1, _id: 0}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "10":
+            //List the name of all the products of Facebook
+            db.collection('companies').find({name: 'Facebook'}, {'products.name': 1}).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result[0]);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "11":
+            //List the people that are working at Facebook right now (check relationships field)
+            db.collection('companies').find({$and: [{name: 'Facebook'}, { "relationships.is_past": true}]}).toArray((error, result) => {
```

Comment:
Por un lado... si trabajan en facebook es que la relationship is NOT past, por lo tanto tendrías que buscar falses. 

De todas formas esta es un poco trampa, si revisas bien las personas que te devuelve verás que hay gente que es "false" aunque has buscado true. Este tipo de query no se puede hacer así. Te sugiero que investigues el operador $filter (si no me equivoco) o que recurras a algo en la línea de:

console.log(result[0].relationships.filter(emp => !emp.is_past))